edx/analytics/tasks/util/s3_util.py

```diff
@@ -135,7 +135,10 @@ def __init__(self, aws_access_key_id=None, aws_secret_access_key=None, **kwargs)
         if not aws_secret_access_key:
             aws_secret_access_key = self._get_s3_config('aws_secret_access_key')
         if 'host' not in kwargs:
-            kwargs['host'] = self._get_s3_config('host') or 's3.amazonaws.com'
+            log.error("S3 'host' not in kwargs")
+            log.error("S3 'host' configured to: %s", self._get_s3_config('host'))
+            kwargs['host'] = self._get_s3_config('host') or 's3.eu-west-1.amazonaws.com'
```

Comment:
@brianhw Do you happen to know where this S3 `host` configuration is pulled from?

I've tried the `override.cfg` and `boto.cfg` files, but the only reliable way to get this to use my `eu-west-1` S3 host is by modifying this line here to change the default.

Reply:
@pomegranited These config items seem to come from luigi. ([luigi config docs here](https://luigi.readthedocs.io/en/stable/configuration.html))

Could be coming from `client.cfg`? That's the only default file I can find that luigi will look at. Could also be overridden with `LUIGI_CONFIG_PATH` somewhere.